### DISCUSSION TITLE
RuinedFooocus on lightning.ai

### DISCUSSION THREAD

**Discussion by NoahChokoh** [runew0lf/RuinedFooocus#195] [Show and tell]
Hello everyone,

If anyone is interested, i could install Ruined Fooocus on lightning.ai.
It is just a basic install following the Linux install commands (i'm not a developper btw).

I share here my configuration if someone is interested. One click install :
https://lightning.ai/naohchokoh/studios/ruinedfooocus?query=ruinedfooocus

I made that studio using a NVIDIA L4 Tensor Core graphic card (GPU) optimized for AI workloads, 16 vCPUs and 64 GB of RAM. It's pretty fast. And the best part, you have 22 hours free every month.

First you will need to create an account on Lightning . Ai
Then wait 2 to 3 days for your email address to be accepted.
After that you can click on my link above to copy my configuration.
To launch the app, click on Port Viewer on the right side, then open in a new tab for a better user experience.

If someone need help, just drop a message.
I'd be happy to help.

:)

**Comment by Greninja143-cmd** [comment]
your studio got deleted can you give me link please

**Comment by Greninja143-cmd** [comment]
your studio got deleted can you help me in installing on lightning ai
